chat/user_conversation.py

```diff
@@ -58,8 +58,7 @@ def delete(self):
         })
 
 
-def total_unread():
-    user_id = current_user_id()
+def total_unread(user_id=current_user_id()):
```

Comment:
the default value of the `user_id` argument is evaluated when the function is declared, but the default value of the `current_user_id()` should depends one the context